packages/default/scss/colorpicker/_layout.scss

```diff
@@ -230,6 +231,11 @@
             }
             &.k-alpha-slider {
                 margin-right: $colorpicker-margin / 2;
+
+                &[dir="rtl"] {
```

Comment:
The `[dir="rtl"]` style selector should be applied one the same level as `k-flatcolorpicker` and **not** on every slider element. Then all the elements that need alignment under RTL mode, should be nested in it. 

```suggestion
             [dir="rtl"] .k-flatcolorpicker {
                 // styles of sliders go here
             }   
```

See the rendering: 
![image](https://user-images.githubusercontent.com/8842674/60575379-17ad7b80-9d84-11e9-9d5f-fca27ab28c0d.png)

Check here, for exampe https://github.com/telerik/kendo-themes/blob/develop/packages/default/scss/notification/_layout.scss#L58